.travis.yml

```diff
@@ -13,17 +13,11 @@ before_install:
     - export HUSKY_SKIP_HOOKS=1
 jobs:
   include:
-    - name: "Commit Lint"
-      script: commitlint-travis
-    - name: "Code Lint"
-      script: npm-run-all clean build:i18n lint
-    - name: "Flow Types"
-      script: npm-run-all clean build:i18n && echo "Running Flow on $(yarn flow ls | wc -l) files" && yarn flow
-    - name: "Unit Tests"
-      script: yarn test
+    - name: "Lint"
+      script: commitlint-travis && npm-run-all clean build:i18n lint && echo "Running Flow on $(yarn flow ls | wc -l) files" && yarn flow
     - name: "E2E Tests"
       script: yarn test:e2e
-    - name: "Build Babel"
-      script: npm-run-all clean build:i18n build:ci:es
+    - name: "Unit Tests && Build Babel"
```

Comment:
How about something like this:

```
    - name: "Lint"
      script: commitlint-travis && npm-run-all clean build:i18n lint && echo "Running Flow on $(yarn flow ls | wc -l) files" && yarn flow
    - name: "Tests"
      script: npm-run-all test test:e2e
    - name: "Build"
      script: npm-run-all clean build:i18n build:ci:es build:ci:dist
```

Reply:
I moved the babel build next to lint, since that one is fast.
e2e dist and tests are the 3 longest running, so take the most time.

Reply:
I now combined linting and build in 1 step since it was still fast, unit and e2e. So 3 jobs now.